2-0-data-structures-and-algorithms/src/main/java/com/bobocode/node/Node.java

```diff
@@ -0,0 +1,12 @@
+package com.bobocode.node;
+
+/**
+ * Class {@link Node} is a very simple data structure that consists of an element itself and the reference to the next
+ * node. A element can have any value since it's a generic. A reference to the next node allows to link {@link Node}
```

Comment:
Typo. "An element"

Reply:
It's not a typo, it's a grammatical mistake :)